mitiq/zne/tests/test_zne.py

```diff
@@ -609,6 +609,12 @@ def test_two_stage_zne(
 
     assert len(circs) == len(scale_factors)
 
+    circs_default_scaling_method = scaled_circuits(
+        frontend_circuit, scale_factors
+    )
+
+    assert len(circs_default_scaling_method) == len(scale_factors)
```

Comment:
This feels unnecessary to me since it's actually a test of a for loop with an append call. If we are going to keep it, it should be in it's own test, not an existing one.

Reply:
@purva-thakre out of curiosity why do you think that the default argument needs a test? It's my first time seeing a test like that, and I'm not really convinced it's needed, but you seem to want to keep it!

Reply:
@natestemen It's to make code coverage happy. In the past, I noticed the default option line would show up as uncovered if there wasn't a unit test for it.